sormas-ui/src/main/java/de/symeda/sormas/ui/samples/SampleCreateForm.java

```diff
@@ -186,7 +186,6 @@ protected void addFields() {
 			defaultValueChangeListener();
 			final NullableOptionGroup samplePurposeField = (NullableOptionGroup) getField(SampleDto.SAMPLE_PURPOSE);
 			samplePurposeField.setValue(SamplePurpose.EXTERNAL);
-			getField(SampleDto.PATHOGEN_TEST_RESULT).setVisible(false);
```

Comment:
I find the layout a little confusing like that, probably because there is the final result field, and two rows below the test result field. 
Usually, when creating a pathogen test while creating a sample, the user has to click a checkbox labelled "Create a test result for this sample now", and only after that the test result field shows. When processing a lab message, the test result field shows right away.
I suggest to add a thin horizontal line here:
![image](https://user-images.githubusercontent.com/67001822/130029004-21700e0c-bb20-4702-96e3-370ead842805.png)
This emphasizes that the final result belongs to the sample, and the test result belong to the pathogen test.

Reply:
Ok, I'll do so.